packages/entities/entities-plugins/src/locales/en.json

```
@@ -805,13 +805,13 @@
             },
             "branch": {
               "name": "branch",
-              "summary": "Conditional execution",
+              "summary": "Execute nodes based on matching input conditions",
               "description": "Execute different nodes based on matching input conditions, such as a cache hit or miss."
             },
             "cache": {
               "name": "cache",
-              "summary": "Fetch cached data",
-              "description": "Fetch data from the cache."
+              "summary": "Store and retrieve data from cache",
+              "description": "Stored data into cache and fetch cached data from cache."
```

```suggestion
              "description": "Store data into cache and fetch cached data from cache."
```

Should it be `Store`?